commit 8b187755d53036bf05dbedb387cefbccbb109913 (refs/changes/35/3635/2)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2018-03-19T19:10:46+00:00 (1 year, 7 months ago)
    
    TOOLS-1302 sdcadm update failed when importing image
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/lib/procedures/shared.js b/lib/procedures/shared.js
index 82f6f5e..4c0751e 100644
--- a/lib/procedures/shared.js
+++ b/lib/procedures/shared.js
@@ -762,7 +762,8 @@ function imgadmInstallRemote(opts, callback) {
 
     var argv = [
         '/opt/smartdc/bin/sdc-oneachnode',
-        format('-n %s ', server),
+        '-n', server,
+        '-T', '300',
         format('/usr/sbin/imgadm import -q %s', img.uuid)
     ];
 
